code/modules/food/cooking_machines.dm

```diff
@@ -170,7 +170,7 @@ var/global/ingredientLimit = 10
 	set category = "Object"
 	set src in oview(1)
 
-	if(isjustobserver(usr))
+	if(isjustobserver(usr) || ismouse(usr))
```

Comment:
At this point, wouldn't if(!isCarbon(usr)) be better?

Because next you're going to have transmogged goliaths removing the reagents and such

Reply:
Silicons should still be able to remove reagents, we have service bots

I'll see if I can do something like if(!(iscarbon(usr) || issilicon(usr)))

Reply:
Mice aren't allowed to remove reagents because mice are made of dead players, and removing the reagents can have a direct effect on the round.

I see no reason why other non-carbon mobs shouldn't be allowed to remove the reagents, though.

Reply:
Fair point. I was under the impression that silicons were mob/living/carbon/silicon, but seems I was mistaken.